.github/workflows/tests.yml

```
@@ -3,12 +3,8 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.event.number }}-${{ github.event.ref }}
   cancel-in-progress: true
 on:  # yamllint disable-line rule:truthy
-  push:
-    branches:
-      - '*'
-  pull_request:
-    branches:
-      - '*'
+  push: {}
+  pull_request: {}
```

Note the meaning is the same; on the other hand, I would suggest running them only against the master to reduce the number of jobs running in parallel the PR comes from the root repo

```suggestion
 push:
    branches: ['master']
  pull_request:
    branches: ['master']
```